### ISSUE TITLE
Value Error 

### ISSUE THREAD

**Issue by maleeham** [lyst/lightfm#487]
<img width="1392" alt="Screen Shot 2019-08-28 at 1 20 58 PM" src="https://user-images.githubusercontent.com/42660845/63894966-8b9a8780-c9b3-11e9-9568-09819b783821.png">

I have trained the model on the same data as:
model = LightFM(loss='logistic')
model.fit(xtrain) 

When I am making the predictions, there is a size mismatch error. Weirdly it works for first 25 users and then shows value error. I need recommendations list for all the users in the train data.

**Comment by SimonCW** [comment]
I’m closing this issue because it has been inactive for a long time. If you still encounter the problem, please open a new issue.

Thank you!
